CRM/Core/BAO/MessageTemplate.php

```diff
@@ -122,7 +122,7 @@ public static function add(&$params) {
   public static function del($messageTemplatesID) {
     // make sure messageTemplatesID is an integer
     if (!CRM_Utils_Rule::positiveInteger($messageTemplatesID)) {
-      CRM_Core_Error::fatal(ts('Invalid Message template'));
+      throw new Exception('Invalid Message template');
```

Comment:
it should be CRM_Core_Exception rather than just Exception

Reply:
@eileenmcnaughton Cheers, I've replaced Exception with CRM_Core_Exception